index.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] As asked by the previous reviewer kindly remove the `Eslint` disabling codes from this file. As explained by him you were given the option to use the `/* eslint-disable max-classes-per-file */ rule`. Anything else besides this can be avoided and it Eslint helps you write a better JavaScript code

Reply:
Thank you so much. I was able to fix it.